docs/requirements.txt

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynamic.grid.calculation@alliander.com>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+sphinx==4.5.0
```

Comment:
maybe you don't need a separate `requirements.txt` at all.

You can specifiy all the doc requirements in `pyproject.toml` under section `[project.optional-dependencies]`. Then you can run `pip install -e .[doc]`.

Reply:
Okay, i think it might be possible to customize all the doc building process by writing it into the `.readthedocs.yaml` file instead of using the settings on the website. 
I'll check that so it can be built this way.